libs/auth/src/machine_ids.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * The machine ID in dev certs
+ */
+export const DEV_MACHINE_ID = '0000';
```

Comment:
suggest a machine ID that actually says DEV

Reply:
Related, we seem to hardcode this value in many places. Can they reference this version instead?

Reply:
I initially did that (in fact, I even went a step further and started by giving every machine type a unique dev machine ID) but then saw how much of the code/tests/snapshots use this existing 0000 ID. I'm now running quite short on time so will likely leave as is.

I noticed the hardcoding outside of this lib and considered changing it. Happy to do that

Reply:
https://github.com/votingworks/vxsuite/pull/5557/commits/e9e280f61942a38408054fb2849fb6d637deed1b